server/webapp/WEB-INF/rails.new/app/presenters/api_v3/admin/environments/environment_config_representer.rb

```diff
@@ -0,0 +1,83 @@
+##########################################################################
+# Copyright 2017 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV3
+  module Admin
+    module Environments
+      class EnvironmentConfigRepresenter < ApiV3::BaseRepresenter
+        alias_method :environment, :represented
+
+        error_representer({
+                            'variables' => 'environment_variables',
+                          })
+
+        link :self do |opts|
+          opts[:url_builder].apiv3_admin_environment_url(name: environment.name)
+        end
+
+        link :doc do |opts|
+          'https://api.gocd.io/#environment-config'
+        end
+
+        link :find do |opts|
+          opts[:url_builder].apiv3_admin_environment_url(name: '__environment_name__').gsub(/__environment_name__/, ':environment_name')
+        end
```

Comment:
You had also shown `self_with_config_repo` and `find_with_config_repo` in the proposed json. Don't forget to add them.